### PR TITLE
Add strong typing for the following API methods

### PR DESCRIPTION
**Pull request by jesusbv**
kiwi/storage/raid_device
kiwi/storage/setup
kiwi/storage/subformat/base
kiwi/storage/subformat/gce
kiwi/storage/subformat/ova
kiwi/storage/subformat/qcow2

This references issue #1644

### CODE REVIEW

File: kiwi/storage/subformat/__init__.py [hunk 1]
```diff
@@ -56,9 +58,8 @@ def new(
             'base': 'Base'
         }
         try:
-            (custom_args, module_namespace) = DiskFormat._custom_args_for_format(
-                name, xml_state
-            )
+            (custom_args, module_namespace) = DiskFormat.\
+                _custom_args_for_format(name, xml_state)
```
Comment (davidcassany) [inline kiwi/storage/subformat/__init__.py:62]: Just out of curiosity. Is this chance done for any specific reason? I am fine with it, just curious.

**Review by davidcassany — APPROVED** [review]
Looks good to me :+1:

File: kiwi/storage/subformat/__init__.py (hunk 1 above)
Comment (schaefi) [reply kiwi/storage/subformat/__init__.py:62]: Only to break up the long line. Do you see an issue with it ?

Comment (davidcassany) [reply kiwi/storage/subformat/__init__.py:62]: No, not at all. It was just curiosity to know if there was any further motivation beyond breaking the line.

**Review by schaefi — APPROVED** [review]

**Pull request merged**